examples/starter/Readme.md

```diff
@@ -0,0 +1,106 @@
+# Example - A basic example to start with the MongoDB Atlas and Terraform
+
+This project aims to provide a very straight-forward example of setting up Terraform with MongoDB Atlas. This will create the following resources in MongoDB Atlas:
+
+- Atlas Project
+- MongoDB Cluster - M10
+- Database User
+- IP Access List
+
+You can refer to the MongoDB Atlas documentation to know about the region names used in MongoDB Atlas respective to the Cloud Provier's region name.
+[Amazon Web Services (AWS)](https://docs.atlas.mongodb.com/reference/amazon-aws/#amazon-aws)
+[Google Cloud Platform (GCP)](https://docs.atlas.mongodb.com/reference/google-gcp/#google-gcp)
+[Microsoft Azure](https://docs.atlas.mongodb.com/reference/microsoft-azure/#microsoft-azure)
+
+## Dependencies
+
+* Terraform v0.13
+* A MongoDB Atlas account - provider.mongodbatlas: version = "~> 0.7.0"
```

Comment:
[small edit] Provider should be on it's on line?

Reply:
updated Readme.md